app/views/v1/legislators/index.jbuilder

```diff
@@ -0,0 +1,6 @@
+if @error
+  json.error(@error)
+else
+  json.target_legislators(@user.target_legislators_json)
+  json.address_required(@user.address_required?)
```

Comment:
For these two lines.  I'd actually move `@user.target_legislators_json` and `@user.address_required?` to the controller action.  Then in the controller have `@target_legislators = @user.target_legislators_json` It just makes it more explicit what you're passing to the template builder.